src/utils/validateNric.tsx

```diff
@@ -1,6 +1,6 @@
 // Thanks to https://gist.github.com/kyrene-chew/6f275325335ab27895beb7a9a7b4c1cb
 
-export const nricRegex = /(\D)(\d{7})(\D)/;
+export const nricRegex = /(\D)(\d{7})(\D)$/;
```

Comment:
Perhap should include `^`, else 12345S8888888W will be accepted too
```js
export const nricRegex = /^(\D)(\d{7})(\D)$/`;
```

Reply:
ohyeahh thanks nice catch!